core/controllers/acl_decorators.py

```diff
@@ -2919,9 +2919,31 @@ def test_can_edit(
             raise self.NotFoundException
         if role_services.ACTION_EDIT_ANY_QUESTION in self.user.actions:
             return handler(self, question_id, **kwargs)
-        else:
+        if (role_services.ACTION_EDIT_QUESTION_IN_MANAGED_TOPIC
+        in self.user.actions):
+            skills = question_services.get_skills_linked_to_question(
+                question_id)
+            if not skills:
+                raise self.UnauthorizedUserException(
+                    'You do not have credentials to edit this question.')
+            skill_ids = set(skill.id for skill in skills)
+            managed_topic_ids = [
```

Comment:
I suggest adding a newline above this, since it's a separate "block" conceptually

Reply:
Done!